flags/flags.go

```diff
@@ -398,6 +398,11 @@ flag exists so it can be configured in $STEPPATH/config/defaults.json.`,
 		Name:  "kms",
 		Usage: "The <uri> to configure a Cloud KMS or an HSM.",
 	}
+
```

Comment:
Is `attest` descriptive enough? Should it be something like `attestation-uri`?

Reply:
Fixed with 4fc5893